test/CheckRequirements.Tests.ps1

```diff
@@ -0,0 +1,45 @@
+BeforeAll {
+    . $PSScriptRoot\Shared.ps1
+}
+
+Describe 'Test-GitVersion' {
+    InModuleScope 'posh-git' {
+        It 'Returns true for Git for Windows newer than 2.15 (<_>)' -ForEach @(
+            'git version 2.33.0-rc2.windows.1',
```

Comment:
@dscho can you confirm this is the correct version format? Your commit mentions `-rc` but the regex was `\.rc`.

Reply:
https://github.com/dahlbyk/posh-git/issues/860#issuecomment-846617267 has `git version 2.32.0.rc1.windows.1`. Pushed a commit to support either. 🤷‍♂️